testsuite/features/secondary/srv_menu.feature

```diff
@@ -116,7 +116,7 @@ Feature: Web UI - Main landing page menu, texts and links
 
   Scenario: The manager configuration submenu of the admin menu
     When I follow the left menu "Admin > Manager Configuration"
-    Then I should see a "SUSE Manager Configuration - General Configuration" text
+    Then I should see a "Configuration - General Configuration" text
```

Comment:
This change seems unrelated, maybe it has been changed in master meanwhile?

Reply:
This is a bugfix: for Uyuni the text shown will be `Uyuni Configuration` - this change will fix the failing testsuite test